Cubical/Foundations/PathSplitEquiv.agda

```diff
@@ -0,0 +1,176 @@
+{-
+
+Theory about path split equivalences. 
+They are convenient to construct localization HITs as in 
+(the "modalities paper")
+https://arxiv.org/abs/1706.07526
+
+- there are construction from and to equivalences ([pathSplitToEquiv] , [equivToPathSplit])
+- the structure of a path split equivalence is actually a proposition ([isPropIsPathSplitEquiv])
+
+The module starts with a couple of general facts about equivalences:
+
+- if f is an equivalence then (cong f) is an equivalence ([equivCong])
+- if f is an equivalence then pre- and postcomposition with f are equivalences ([preCompEquiv], [postCompEquiv])
+
+(those are not in 'Equiv.agda' because they need Univalence.agda (which imports Equiv.agda))
+-}
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Foundations.PathSplitEquiv where
+
+open import Agda.Primitive
```

Comment:
Why is this needed?

Reply:
For the max of universe levels.

Reply:
This is already exported by Cubical.Core.Everything and it's called `ℓ-max`. Maybe this is a bad name, but let's use it for now.